### ISSUE TITLE
conflict with forward slash or directory

### ISSUE THREAD

**Issue by tempral** [h5p/h5p-editor-php-library#211]
<img width="554" alt="Screenshot 2024-02-15 125341" src="https://github.com/h5p/h5p-editor-php-library/assets/11719067/1c368930-84e6-4836-8d63-5293d5c1d9be">

when i try to automatize change the content (manually or programmatically) inside the h5p archive
and later try to import by uploading it,
it fails as explained in the attached image

if i manipulate the h5p archive with 7zip, it succeed

so I guess it is merely a / or \ difference?

**Comment by otacke** [comment]
@tempral Hi! I was just made a community maintainer for H5P Group's repositories. I am now trying to tackle all the issues and pull requests that are still open.

First of all, I am terribly sorry that you have never received a reply.

If you manually zip files, make sure to use the flags that H5P expects: -D (do not add directory entries) and -X (eXclude eXtra file attributes), e.g. as:

```
zip -rDX myNewFile.h5p *
```

or to configure your zip tool accordingly. 7zip might set these by default, so it would work there.

Please close the issue if that answers your question (even though late).

**Comment by otacke** [comment]
@tempral Given that you have not answered yet, I am going to close this issue in about a week unless you holler back.

**Comment by tempral** [comment]
I haven't create other zip files since then. Now I don't have a computer with me to test your suggested solution .
My intention was just to let know this behavior, in case you want to support both directory character "/" and "\\"

**Comment by otacke** [comment]
I don't think this is related to the direction of the slash. It's the flags set when zipping that matter. It's the directory entries inside the zip file that cause the trouble, and the `-D` flag ensures the entries are not recorded.

I'll close this issue then, but please feel free to open another one if you encounter something similar.
